wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Extensions.kt

```diff
@@ -22,23 +22,38 @@ import kotlin.jvm.JvmStatic
 class Extensions private constructor(
   val location: Location,
   val documentation: String,
-  val start: Int,
-  val end: Int
+  val values: List<Any>
```

Comment:
document what the potential values are?

Reply:
I guess that’s implied on line 31

Reply:
ahh, found it in the element

Reply:
Yeah, it wrote it the same way `ReservedElement` was documented.